examples/solid-start/src/utils/grafbase.ts

```diff
@@ -0,0 +1,3 @@
+import { GraphQLClient } from 'graphql-request'
+
+export const grafbase = new GraphQLClient('http://127.0.0.1:4000/graphql')
```

Comment:
Shouldn't this be a env variable?

Reply:
This should be an env variable and also configure the header x-api-key.

Reply:
Right 👌